### PR TITLE
feat: fw-2570 add schedule tip

### PR DESCRIPTION
**Pull request by swkatmask**
https://mask.atlassian.net/browse/fw-2570

### CODE REVIEW

File: src/store/usePreferenceStore.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    SHOW_SCHEDULE_POST_TIP: boolean;
+}
+
+const defaultPreferences = {
+    SHOW_SCHEDULE_POST_TIP: true,
+};
+
+export interface PreferencesState {
+    preferences: Preferences;
+    getPreference(key: keyof Preferences): unknown;
```
Comment (guanbinrui) [inline src/store/usePreferenceStore.ts:17]:
```suggestion
    getPreference<T extends keyof Preferences>(key: T): Preferences[T];
```

File: src/store/usePreferenceStore.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import { createJSONStorage, persist } from 'zustand/middleware';
+import { immer } from 'zustand/middleware/immer';
+
+import { createSelectors } from '@/helpers/createSelector.js';
+
+interface Preferences {
+    SHOW_SCHEDULE_POST_TIP: boolean;
+}
+
+const defaultPreferences = {
```
Comment (guanbinrui) [inline src/store/usePreferenceStore.ts:11]:
```suggestion
const defaultPreferences: Preferences = {
```

File: src/components/Compose/ScheduleTaskList.tsx [hunk 3]
```diff
@@ -79,7 +83,7 @@ const ScheduleTaskItem = memo(function ScheduleTaskItem({ task, index }: { task:
     }, [task.uuid]);
 
     return (
-        <div className="border-b border-line py-3">
+        <div className="border-b border-line p-3">
```
[the remark this answers is not in the document]
Comment (swkatmask) [reply src/components/Compose/ScheduleTaskList.tsx:86]: Yes, px is needed.

[the remark this answers is not in the document]
Comment (swkatmask) [reply src/components/Compose/ScheduleTaskList.tsx:86]: ## On staging right now
image

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**